python/test/t_wingweight_function.py

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env python
+
+import openturns as ot
+from openturns.testing import assert_almost_equal
+from openturns.usecases import wingweight_function
+
+ot.TESTPREAMBLE()
+ot.PlatformInfo.SetNumericalPrecision(5)
```

Comment:
this would only affect printing results, which is not done here and can be dropped

Reply:
Thanks. it is now fixed